src/style.css

```
@@ -0,0 +1,28 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars > :not(:last-child) {
```

don't increase specificity if it's unnecessary
```suggestion
stars__star:not(:last-child) {
```